custom_components/innova/options_flow.py

```diff
@@ -0,0 +1,30 @@
+from homeassistant import config_entries
+import voluptuous as vol
+from datetime import timedelta
+
+from .const import DOMAIN
+
+
+class InnovaOptionsFlowHandler(config_entries.OptionsFlow):
+    def __init__(self, config_entry: config_entries.ConfigEntry):
+        """Initialize the Innova options flow handler."""
+        self.config_entry = config_entry
+
+    async def async_step_init(self, user_input=None):
+        """Manage the options for the Innova integration."""
+        if user_input is not None:
+            # Store the update interval in seconds in Home Assistant's data
+            self.hass.data["innova_update_interval_seconds"] = user_input["update_interval_seconds"]
+            return self.async_create_entry(title="", data=user_input)
+
+        # Default to 300 seconds (5 minutes) if not set
+        current_interval = self.config_entry.options.get("update_interval_seconds", 300)
```

Comment:
Here the default is 300, but in `__init__.py`, it was 600. We should probably choose one and keep it consistent everywhere.
Since the value was already hardcoded at 600, I think it might be a good idea to keep that default, but I'm on 300 would probably be OK too.

Reply:
Thanks for the feedback I missed that they are not consistent now, I've removed the default value to a const in order to prevent this in the future.